Backend/Backend/Models/Sponsor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Backend.Models;
+
+public class Sponsor : BaseEntity
+{
+
+    public Sponsor() 
+    {
+        Name = string.Empty;
+        Website = string.Empty;
+    }
+
+    [Key]
+    public Guid Id { get; set; }
+    
+    public string Name { get; set; }
+    
+    public string Website { get; set; }
```

Comment:
This can be a nullable, some people will not have a website potentially. While it does get init, it shouldnt be an issue when it does not like in tests